bin/schema_generator.sh

```diff
@@ -55,6 +50,10 @@ function setup_git_ssh() {
     chown -R "$(id -u):$(id -g)" "$HOME/.ssh"
     chmod 700 "$HOME/.ssh"
     chmod 700 "$HOME/.ssh/id_ed25519"
+
+    # add private key to the ssh agent to prompt for password once
```

Comment:
I'm assuming this is a noop on CI in that the ssh key doesn't have a passphrase.

Reply:
This should be the case. This also only runs for the duration of the script session.